docs/faq.md

```diff
@@ -110,6 +110,7 @@ The syntax of the mode string is like this:
 * `me_car-r` is for car rental (like Budget)
 * `me_car-p` is for car pooling (like BlaBlaCar)
 * `me_mot` is for your own motorbike
+* `me_mic` is for your own micro-mobility (see `micVehicleType` input for `routing.json`)
```

Comment:
@nighthawk : just wondering, would it be better to model this as `modes=me_mic[_<vehicleType>]` instead of `modes=me_mic&micVehicleType=<vehicleType>` ? It makes sense to me as it's very similar to `pt_pub_<fine-grained mode>`. Also it'd easily allow to request 2 different micromobility types in the same request.

Reply:
i second this, as it can be used in the response, too.

alternatively you could do `modes=me_mic` and `avoidModes=me_mic_bic`, just like `pt_pub`?

Reply:
Good idea, @JulietaMarcos, let's go with what you suggest.